<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="friendName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FriendName</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addOrDelButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加/删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:url(:/resource/BG4.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="searchFailed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发现你的狐朋狗友：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主界面</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>searchText</zorder>
   <zorder>searchButton</zorder>
   <zorder>friendName</zorder>
   <zorder>addOrDelButton</zorder>
   <zorder>searchFailed</zorder>
   <zorder>title</zorder>
   <zorder>returnButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="general.qrc"/>
 </resources>
 <connections/>
</ui>
